waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -171,10 +149,15 @@ method register*(
 ): Future[void] {.async: (raises: [Exception]).} =
   initializedGuard(g)
 
-  await g.registerBatch(@[rateCommitment])
+  try:
+    let leaf = rateCommitment.toLeaf().get()
+    await g.registerBatch(@[leaf])
+  except CatchableError:
+    raise newException(ValueError, getCurrentExceptionMsg())
```

Comment:
```suggestion
    raise newException(ValueError, "exception reraised in atomicBatch: " & getCurrentExceptionMsg())
```

Reply:
as @alrevuelta mentioned in [thread](https://discord.com/channels/1110799176264056863/1246012979519160392/1247571125475479613), we will ask operators to re-register memberships to the new contract